### ISSUE TITLE
Spaces in URLs are not percent encoded

### ISSUE THREAD

**Issue by kaimac1** [Digital-Sapphire/PyUpdater#10]
My application name contains spaces. When trying to download the zip containing the update, PyUpdater makes the following (bad) request:

```
"GET /updates/App name-win-1.1.zip HTTP/1.1" 400
```

The correct request would be:

```
"GET /updates/App%20name-win-1.1.zip HTTP/1.1" 200
```

**Comment by JMSwag** [comment]
Fixed in d6892bdf88ffbdf94c215b80db86c6881d41d4f8

Please install from master until the next pypi release.

**Comment by JMSwag** [comment]
@ksrm v2.1.3 on pypi includes this fix.
